docker/images/pinot-thirdeye/bin/start-thirdeye.sh

```diff
@@ -18,32 +18,64 @@
 # under the License.
 #
 
+# Script Usage
+# ---------------------------------------------
+# ./start-thirdeye.sh ${CONFIG_DIR} ${MODE}
+#
+# - CONFIG_DIR: path to the thirdeye configuration director
+# - MODE: Choices: {frontend, backend, * }
+#       frontend: Start the frontend server only
+#       backend: Start the backend server only
+#       For any other value, defaults to starting all services with an h2 db.
+#
+
 if [[ "$#" -gt 0 ]]
 then
   CONFIG_DIR="./config/$1"
 else
   CONFIG_DIR="./config/default"
 fi
 
-echo "Starting H2 database server"
-java -cp "./bin/thirdeye-pinot.jar" org.h2.tools.Server -tcp -baseDir "${CONFIG_DIR}/.." &
-sleep 1
 
-echo "Creating ThirdEye database schema"
-java -cp "./bin/thirdeye-pinot.jar" org.h2.tools.RunScript -user "sa" -password "sa" -url "jdbc:h2:tcp:localhost/h2db" -script "zip:./bin/thirdeye-pinot.jar!/schema/create-schema.sql"
+function start_server {
+  class_ref=$1
+  config_dir=$2
+  java -Dlog4j.configurationFile=log4j2.xml -cp "./bin/thirdeye-pinot.jar" ${class_ref} "${config_dir}"
+}
 
-if [ -f "${CONFIG_DIR}/bootstrap.sql" ]; then
-  echo "Running database bootstrap script ${CONFIG_DIR}/bootstrap.sql"
-  java -cp "./bin/thirdeye-pinot.jar" org.h2.tools.RunScript -user "sa" -password "sa" -url "jdbc:h2:tcp:localhost/h2db" -script "${CONFIG_DIR}/bootstrap.sql"
-fi
+function start_frontend {
+  echo "Running Thirdeye frontend config: ${CONFIG_DIR}"
+  start_server org.apache.pinot.thirdeye.dashboard.ThirdEyeDashboardApplication "${CONFIG_DIR}"
+}
+
+function start_backend {
+  echo "Running Thirdeye backend config: ${CONFIG_DIR}"
+  start_server  org.apache.pinot.thirdeye.anomaly.ThirdEyeAnomalyApplication "${CONFIG_DIR}"
+}
+
+function start_all {
```

Comment:
maybe add a **start_database** function for h2 specifically, so it can be started just like front and backend. imo otherwise **start_all** has an "unexpected" side-effect of bringing up a db

Reply:
H2 *does* start in the background on line 58. The other calls on line 62 and 66 create the schema and bootstrap some metadata, respectively.